### PR TITLE
URI instead of URL

### LINKED ISSUE #1690: build failed on java 21+
`URL(java.lang.String) in java.net.URL has been deprecated`

### PR DESCRIPTION
**Pull request by h1alexbel**
closes #1690

### CODE REVIEW

File: src/test/java/org/cactoos/io/InputOfTest.java [hunk 1]
```diff
@@ -54,8 +55,8 @@
 /**
  * Test case for {@link InputOf}.
  *
- * @since 0.1
  * @checkstyle ClassFanOutComplexityCheck (500 lines)
+ * @since 0.1
  */
 @SuppressWarnings({"PMD.TooManyMethods", "PMD.ExcessiveImports", "unchecked"})
```
Comment (pnatashap) [inline src/test/java/org/cactoos/io/InputOfTest.java:61]:
```suggestion
@SuppressWarnings({"PMD.TooManyMethods", "PMD.ExcessiveImports", "unchecked", "PMD.JUnitTestsShouldIncludeAssert"})
```

Comment (pnatashap) [reply src/test/java/org/cactoos/io/InputOfTest.java:61]: And the same fix in the next text class will fix codacy error

**Review by pnatashap — CHANGES_REQUESTED** [review]
@h1alexbel  Codacy errors should be fixed

**Comment by h1alexbel** [comment]
@pnatashap thanks for your suggestions, fixed. Take a look, again please

**Review by pnatashap — APPROVED** [review]

**Comment by pnatashap** [comment]
@yegor256 please take a look. I think it is needed to support jdk 21+

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**